### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by TheBeardedLlama**
fix typo

### CODE REVIEW

**Comment by Eilon** [comment]
@javiercn - can you merge?

**Comment by TheBeardedLlama** [comment]
apologies for the two word title

I was reading the docs and came across the typo, so just clicked edit and went through the motions

hopefully it didn't cause offence :)

**Comment by Eilon** [comment]
@TheBeardedLlama not at all, the title is very clear, so thank you 😄

**Comment by javiercn** [comment]
@Eilon This is being merged into master. Shouldn't be dev the branch to merge it into?

**Comment by Eilon** [comment]
Oh yeah to `dev` only please. Can you move the commit over?

**Comment by TheBeardedLlama** [comment]
repointed, but it added another file to the commit...

let me see if I can remove it

File: global.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-    "projects": ["src", "test/WebSites", "samples"]
```
Comment (Eilon) [inline global.json]: Yeah definitely remove this file from the commit.

**Comment by TheBeardedLlama** [comment]
not sure what happened...

I'll abandon and do a new PR directly for dev branch

apologies @Eilon and @javiercn

**Comment by TheBeardedLlama** [comment]
doh! doc comments are already fixed in dev... :'(

apologies for wasting your time guys

**Comment by Eilon** [comment]
@TheBeardedLlama ah no worries! We're actually going to clean up some of the old branches that are no longer used, which should help remove some of the confusion!

**Pull request closed without merging**